tests/x11/evolution/evolution_prepare_servers.pm

```diff
@@ -75,13 +82,15 @@ sub run() {
 
     # configure postfix
     assert_script_run "postconf -e 'smtpd_use_tls = yes'";
+    assert_script_run "postconf -e 'smtpd_tls_security_level = encrypt'" if (is_sle('>=16') || is_tumbleweed);
```

Comment:
```suggestion
    assert_script_run "postconf -e 'smtpd_tls_security_level = encrypt'" unless (is_sle('<16'));
```
Perhaps it's a good idea to remain consistent with the upper if block.

Maybe it would help to refactor a variable like

```perl
my $dovecot24 = !is_sle('<16');
```
Or something similar? :thinking: